engine/chunk/src/main/java/io/deephaven/chunk/util/hashing/ObjectChunkIdentityEquals.java

```diff
@@ -250,7 +251,7 @@ public void andEqualPairs(IntChunk<ChunkPositions> chunkPositionsToCheckForEqual
 
     // region eq
     static private boolean eq(Object lhs, Object rhs) {
-        return lhs == rhs;
+        return ObjectComparisons.eq(lhs, rhs);
```

Comment:
This is wrong.

Reply:
Good catch. Fixed. Surprised we didn't have any failing test.